tests/features/environment.feature

```diff
@@ -0,0 +1,41 @@
+
+ Scenario: environments$ should return all available environments
+   Given API Sandbox Service with environments$ method
+   And   several registered environments
+   When  user subscribes to environments$ method with a valid EnvironmentsRequest
+   Then  user receives all available environments
+
+  Scenario: selectEnv should select the provided environment
+   Given API Sandbox Service with selectEnv method
+   And   user received several environments from environments$ subscription
+   When  user calls selectEnv method by providing the name of one of these environments
+   Then  the provided environment is selected
+
+  Scenario: Call selectEnv with the name of an unexisting environment
+   Given API Sandbox Service with selectEnv method
+   And   user received several environments from environments$ subscription
+   When  user calls selectEnv method by providing an unexisting environment name
+   Then  a relevant error is returned
+
+  Scenario: Call selectEnv with invalid environment name
+   Given API Sandbox Service with selectEnv method
+   And   user received several environments from environments$ subscription
+   When  user calls selectEnv method by providing one of the following <envName>
+     	 |envName        |
+         |null           |
+         |undefined      |
+         |123            |
+         |'test'         |
```

Comment:
Actually, `'test'` could be a valid envName

Reply:
fixed